sw/device/silicon_creator/lib/sigverify/ecdsa_p256_verify.h

```diff
@@ -0,0 +1,59 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_SIGVERIFY_ECDSA_P256_VERIFY_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_SIGVERIFY_ECDSA_P256_VERIFY_H_
+
+#include "sw/device/silicon_creator/lib/drivers/hmac.h"
+#include "sw/device/silicon_creator/lib/drivers/lifecycle.h"
+#include "sw/device/silicon_creator/lib/error.h"
+#include "sw/device/silicon_creator/lib/sigverify/ecdsa_p256_key.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+enum {
+  /**
+   * A non-trivial constant chosen such that `kSigverifySpxSuccess ^
+   * kSigverifyEcdsaSuccess = kSigverifyFlashExec`.
+   */
+  kSigverifyEcdsaSuccess = 0x2f06b4e0,
```

Comment:
should the comment describe how this was generated?

Reply:
Added reference to .c comments, and updated description under `kSigverifyShares`.